apps/docs/src/development/index.md

```diff
@@ -53,3 +59,80 @@ import { OnyxButton } from "sit-onyx";
   <OnyxButton label="Click me" />
 </template>
 ```
+
+You can check all onyx components and their documentation in our [Storybook](https://storybook.onyx.schwarz).
+
+## First steps <Badge text="recommended" />
+
+Here are some further steps we recommend to take a look at when setting up your new application:
+
+### Layout components
+
+onyx provides [layout components](https://storybook.onyx.schwarz/?path=/docs/layout-about-layouts--docs) for easily setting up common layouts for the app, pages etc. which also take care of ensuring correct scroll containers etc.
+
+Here is an example how a default application looks like when using layout components:
```

Comment:
I had the idea that maybe we could make use of the playground to also let users directly jump and look how it is rendered.
```suggestion
Here is [an example](https://playground.onyx.schwarz/#eNp9UsFOwzAM/ZUoXNf2AKdRkBjaAQTbBFyQIqGo9bqMNomStGya+u84abt10rRb/Pz8/GznQJ+0jpsa6JSmNjNCO1JyWTww6iyjxIKr9SOTotLKOHIgS7nfYckb36vaTUK44gWM4wVvZtyQlqyNqgijVrhIIc7oPZNp0nVBTQwcVLrkDjAiJD3TDhCCA4XcyKDb4z2978W1jiSvAG2/732E1pNBIRl16aFz2yfJXDQkK7m1KOQtR4UReZQp6biQYBh9/Fa1IRpLiUdBOrIBA3EcpwkWH1teaNCB4/lGzugE942Ka1HEW6sknuPgixjNVKVFCWapnVASbzIlIeNzvCzV32vAnKlhMuDZBrLfC/jW4hGm+FgZsGAaYPSYc9wU4Lr0/HMBO3wfk5XK6xLZV5IfYFVZe48dbVbLHG2PeMHtS/hIQhZfdr7D/dlhKG/UM9vAZxT/5POV0U92b+O7UMdki1v8acB4TVygX6x1tP0HmYr25w==) how a default application looks like when using layout components:
```

but somehow the nav bar has a bigger padding than the page and it looks weird 😮  is it just because it looks different in the playground or would it actually look like htat in a "normal" vue app?

Reply:
oooh it's because of media queries..

![image](https://github.com/SchwarzIT/onyx/assets/151019977/ca1d2a23-f654-41c6-9ebf-2b68ca3b55b9)

they only affect the content, not the nav bar

Reply:
Oh, no thats a bug inside the OnyxNavBar

Reply:
you can just create a bug ticket, don't have to solve it in this pr :)

Reply:
#1279